src/audio/mixin_mixout.c

```diff
@@ -1020,6 +1111,73 @@ static int mixout_get_attribute(struct comp_dev *dev, uint32_t type, void *value
 	return 0;
 }
 
+static int mixin_set_large_config(struct comp_dev *dev, uint32_t param_id, bool first_block,
+				  bool last_block, uint32_t data_offset_or_size, char *data)
+{
+	struct ipc4_mixer_mode_config *cfg;
+	struct mixin_data *mixin_data;
+	int i;
+	uint32_t sink_index;
+	uint16_t g;
```

Comment:
if you make a new iteration of this PR - maybe give this variable a longer name, e.g. `gain`. Short names are inconvenient to search for

Reply:
Done.